stripe/object_classes.py

```diff
@@ -26,6 +26,7 @@
     api_resources.Coupon.OBJECT_NAME: api_resources.Coupon,
     api_resources.CreditNote.OBJECT_NAME: api_resources.CreditNote,
     api_resources.Customer.OBJECT_NAME: api_resources.Customer,
+    api_resources.CustomerBalanceTransaction.OBJECT_NAME: api_resources.CustomerBalanceTransaction,
```

Comment:
err, @ob-stripe , do we need this?

Reply:
Why wouldn't we need that line?

Reply:
oh wow I think it's missing in prod, this is a real bug.

Reply:
yeah I think that's correct 😬